### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/knative-pkg/commit/9b854438c489b969e51ead9e3da9e071fa5eafdd#commitcomment-43154258

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/knative-pkg/commit/9e77c641386a138bee94288e8d7870f8fd4dfa6c

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: apis/duck/verify_test.go [hunk 1]
```diff
@@ -361,7 +361,7 @@ func (f *String) Populate() {
 var emptyStringable Stringable
 
 // For testing this doubles as the 'Implementable'
-// and 'Populataable'
+// and 'Populatable'
```
Comment (jsoref) [inline apis/duck/verify_test.go:364]: not terribly confident about this spelling.

File: client/injection/apiextensions/reconciler/apiextensions/v1/customresourcedefinition/reconciler.go [hunk 2]
```diff
@@ -179,7 +179,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 
 	// Initialize the reconciler state. This will convert the namespace/name
-	// string into a distinct namespace and name, determin if this instance of
+	// string into a distinct namespace and name, determine if this instance of
```
Comment (jsoref) [inline client/injection/apiextensions/reconciler/apiextensions/v1/customresourcedefinition/reconciler.go:182]: (this code appears in a number of places, let me know where to change it...)

File: tracker/enqueue_test.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func TestHappyPathsExact(t *testing.T) {
 
 	thing2 := &Resource{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "reffer.knative.dev/v1alpha1",
+			APIVersion: "refer.knative.dev/v1alpha1",
```
Comment (jsoref) [inline tracker/enqueue_test.go:61]: very worried about anything with the word `api`, but I'm pretty sure this is just a test.

File: webhook/psbinding/table_test.go [hunk 4]
```diff
@@ -1518,7 +1518,7 @@ func TestBaseReconcile(t *testing.T) {
 							Type:   "Ready",
 							Status: "False",
 							Reason: "SubjectUnavailable",
-							// prefix comes from apiserrs.NewForbidden
+							// prefix comes from apierrs.NewForbidden
```
Comment (jsoref) [inline webhook/psbinding/table_test.go:1521]: ?

**Review by jsoref — COMMENTED** [review]
Fixes are generally selected automatically by Google Sheets. It's definitely possible that I've messed up, but I've tried to make sure everything is reasonable...

File: hash/hash_test.go [hunk 5]
```diff
@@ -38,10 +38,10 @@ func ExampleChooseSubset_selectOne() {
 	ret := ChooseSubset(tasks, 1, "my-key1")
 	fmt.Println(ret.UnsortedList()[0])
 
-	ret = ChooseSubset(tasks, 1, "somthing/another-key")
+	ret = ChooseSubset(tasks, 1, "something/another-key")
 	fmt.Println(ret.UnsortedList()[0])
 	// Output: task3
-	// task1
+	// task2
```
Comment (jsoref) [inline hash/hash_test.go:44]: afaict, the change of the spelling of the subset changes how the hash plays around and thus which task one gets?

Someone will need to consider this much more carefully than me.

**Review by vagababov — COMMENTED** [review]
/lgtm
/approve

**Comment by vagababov** [comment]
Those seem reasonable

**Pull request merged**